### ISSUE TITLE
scan_config property names

### ISSUE THREAD

**Issue by caseyjlaw** [demorest/evla_mcast#7]
I'm creating a scan config objects and got confused by some of the property names. I interpreted scan_config.npol as the number of pols, but it seems to refer to a pulsar mode (and npol can change between subbands, so I should have known better anyway).
It may help to go over the properties on the top level and per subband for clarity.

**Comment by demorest** [comment]
Yes, you're right.. The naming of ScanConfig.npol is a holdover from when the code's only job was to control the pulsar system (this setting tells the pulsar sofware how many poln products to compute).  The npol you want is indeed a per-subband property (SubBand.npp).  It would be good to more clearly identify things.  Would just prefixing all the pulsar properties with "psr_" be OK with you?  ie npol turns into psr_npol, etc.  Another alternative would be to make a PulsarConfig class to hold those, but I don't know if that's really necessary.

**Comment by caseyjlaw** [comment]
Sure, I think a psr_ prefix would help.
Do you think there are other properties that could be added at the same time? For example, many of the spectral/pol properties require iterating over the get_subband method. It might be nice to automatically populate a few, like the using a property "skyfreqs" to hold a list of sky frequencies per subband.
